src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -125,6 +125,15 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 		startupCommand = common.ExtendPathForCommand(startupCommand, gen.SourcePath)
 	}
 
+	scriptBuilder.WriteString("if [ -f ./" + common.ManifestFileName + " ]; then\n")
+	scriptBuilder.WriteString("    isInManifest=$(cat " + common.ManifestFileName + " | grep -c 'NODE_OPTIONS')\n")
```

Comment:
what does having node_options in the file mean? looks liek we're use a property called NODE_OPTIONS to indicate that app insights should be inject, which is not clear. We could use something like `InjectAppInsights=true` in the manifest.